tsconfig.webpack.json

```diff
@@ -11,6 +11,7 @@
     "importHelpers": true,
     "strictNullChecks": false,
     "downlevelIteration": true,
+    "skipLibCheck": true,
```

Comment:
`react-query`'s builtin types require TypeScript >= 4.1 but our version of angular requires Typescript <= 3.8. I couldn't quickly figure out how to omit types for one dependency (`declare module ...` doesn't work on its own).

Hate sacrificing type checking our libraries but this is necessary unless we can work around this.